### ISSUE TITLE
Dithering setting always resets

### ISSUE THREAD

**Issue by epsilon-a** [Maassoft/ColorControl#420]
Ever since the newest update, the dithering setting doesn't stick and doesn't get applied when applying a preset which uses it. I have also set the registry key.
Was never an issue beforehand.

**Comment by Maxavelli** [comment]
Yes this has definitely become a consistently persistent issue since the big UI update all those patches ago. Of course I can appreciate that it isn't a great priority in the grand scheme of things if they have to prioritize more important bugs & and issues since it still works to set dithering. Even if it's gotten a bit old always having to set it every damn time I change any setting.

However if it's not a particularly difficult or time consuming fix it would be so so soooo awesome if they addressed this in the next patch.
